Readme.md

```diff
@@ -151,6 +151,22 @@ to fix those issues.
 
 Wii network certificates must be extracted from a Wii IOS. A guide for that can be found [here](https://wiki.dolphin-emu.org/index.php?title=Wii_Network_Guide).
 
+## Debugging
+
+To use a debugger with Dolphin (generate backtraces, set breakpoint etc.), it must be built in debug mode `CMAKE_BUILD_TYPE=Debug`. Instead of `cmake`, run `cmake -D CMAKE_BUILD_TYPE=Debug` when following the above build steps.
+
+Dolphin is non-deterministic in dual core mode. Therefore, it's typically recommended to disable this in Dolphin's configuration menu when debugging.
+
+### Linux
+
+`sudo gdb dolphin`
+
+### macOS
+
+gdb is also available for MacOS but might cause problems. Therefore, it's recommended to use lldb.
```

Comment:
You should be consistent with capitalization. This should say "macOS" rather than "MacOS".